221. Maximal Square.py

```diff
@@ -0,0 +1,14 @@
+#space: O (1)
```

Comment:
How come the Space complexity is O(1) when you have created a DP array? Pls be careful while analyzing Time and Space complexities